### PR TITLE
(feat) Pgweb spike

### PR DESCRIPTION
**Pull request by michalc**

### CODE REVIEW

File: pgweb/pgweb-config.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
+import itertools
+import os
+import re
+
+
+def normalise_environment(key_values):
```
Comment (allait) [inline pgweb/pgweb-config.py]: Do we need this function here? We're only using `DATABASE_DSN` from the environment, or am I missing something?

File: pgweb/start.sh [hunk 2]
```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+set -e
+
+python3 pgweb-config.py
```
Comment (allait) [inline pgweb/start.sh]: `export DATABASE_URL=$(python3 pgweb-config.py)` and skip writing the env file entirely?

File: pgweb/pgweb-config.py (hunk 1 above)
Comment (michalc) [reply pgweb/pgweb-config.py]: Ah, DATABASE_DSN isn't the environment variable, it's `DATABASE_DSN_<some-name>`, with (in this context) `<some-name>` not being known

This is set at https://github.com/uktrade/data-workspace/blob/master/dataworkspace/dataworkspace/apps/applications/spawner.py#L170

(It is a bit overcomplex for the case of a single database, but is coded up for more than one...)

File: pgweb/start.sh (hunk 2 above)
Comment (michalc) [reply pgweb/start.sh]: Sounds good!

**Pull request merged**